python/fledge/services/core/api/browser.py

```diff
@@ -65,9 +65,11 @@
 def setup(app):
     """ Add the routes for the API endpoints supported by the data browser """
     app.router.add_route('GET', '/fledge/asset', asset_counts)
+    app.router.add_route('GET', '/fledge/asset/timespan', asset_timespan)
```

Comment:
FYI, if in system with name 'timespan' asset code exists.(possibile scenario) We never get its detail for which we have recorded readings. 
So If user has a request with `/fledge/asset/timespan` - API returns the timespan of the buffered readings for each asset.

We may add any sting literal to the endpoint to distinguish between.
For example: `/fledge/asset/{ANY_MEANINGFUL_LITERAL}/timespan`